assets/pages/digital-subscription-checkout/helpers/paymentProviders.js

```diff
@@ -103,11 +102,15 @@ function showPaymentMethod(
     case 'DirectDebit':
       dispatch(openDirectDebitPopUp());
       break;
+    case null:
+    case undefined:
+      console.log('Undefined payment method');
+      break;
     default:
       console.log(`Unknown payment method ${paymentMethod}`);
```

Comment:
not for this PR but as these are runtime errors we might wanna send them to sentry